html/pfappserver/lib/pfappserver/Form/Config/Profile.pm

```diff
@@ -69,6 +69,25 @@ has_field 'filter.contains' =>
    widget_wrapper => 'DynamicTableRow',
   );
 
+=head2 filter_match_style
+
+The field for filter_match_style
```

Comment:
"Field defining how the configured filters will be applied for matching" would be better.
Otherwise, simply get rid of the comment... it is kind of a "joke" IMO